java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/AlreadyRegisteredException.java

```diff
@@ -8,12 +8,19 @@
  * plug-in, or object adapter more than once for the same ID.
  */
 public final class AlreadyRegisteredException extends LocalException {
+    /**
+     * Creates an AlreadyRegisteredException with the specified kind of object and ID.
```

Comment:
Seems Java uses `Constructs an ...` not sure if we picked `Creates an ...` for Ice or we want to use the Java defaults.

Reply:
I'm fine with either. Java is not consistent (which I think is also fine).